networks/bitcoincashtestnet/info.json

```diff
@@ -27,7 +27,7 @@
     "networkCode": "bitcoincashtestnet",
     "networkName": "Bitcoin Cash Testnet",
     "regex": {
-        "address": "^([13][a-km-zA-HJ-NP-Z1-9]{25,34})|^((bitcoincash:)?(q|p)[a-z0-9]{41})|^((BITCOINCASH:)?(Q|P)[A-Z0-9]{41})$"
+        "address": "^([123][a-km-zA-HJ-NP-Z1-9]{25,34})|^((bitcoincash:)?(q|p)[a-z0-9]{41})|^((BITCOINCASH:)?(Q|P)[A-Z0-9]{41})$"
```

Comment:
Prefix should be `bchtest`.
We could also add testing using the test vectors from the spec: https://bch.info/en/specifications

Reply:
Also max length should be 35 for legacy addresses:
https://bitcoin.stackexchange.com/questions/36944/what-are-the-minimum-and-maximum-lengths-of-a-mainnet-bitcoin-address